### PR TITLE
add one test for imagescan to avoid coverage check failure

### PR DESCRIPTION
**Pull request by matthyx**

### CODE REVIEW

**Comment by matthyx** [comment]
I don't understand this CI failure :/

**Comment by VaibhavMalik4187** [comment]
> I don't understand this CI failure :/

Me neither, I tried running the same command locally but twice:
1. With coverage collection -> Failed
2. Without coverage collection -> Passed

**Comment by matthyx** [comment]
> 1. With coverage collection -> Failed
> 2. Without coverage collection -> Passed

none fail on mine :/

File: core/core/patch_test.go [hunk 1]
```diff
@@ -1,15 +0,0 @@
...
-
-import (
-	"os"
-	"testing"
-
-	"github.com/stretchr/testify/assert"
-)
-
-// sets os.Stdout and os.Stderr to nil
-func TestSetsOsStdoutAndStderrToNil(t *testing.T) {
```
Comment (matthyx) [inline core/core/patch_test.go:11]: @VaibhavMalik4187 it was your test that broke coverage... for still unknown reasons

Comment (VaibhavMalik4187) [reply core/core/patch_test.go:11]: Ahh okay, I should've been more careful. Thanks for the fix.

**Comment by matthyx** [comment]
@Daniel-GrunbergerCA success !!!!

**Pull request merged**